### DISCUSSION TITLE
Width broken on user page

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21742] [New to GitHub]
<p>become of some tweak in the last day width is now broken on user pages max width on the page is correct but the SVG contributions are borked and dont scale when you shrink the page leading to scrollbars</p>

**Comment by ghost** [comment, marked as answer]
<p>Here is a workaround:</p>
<pre><code>body {
   min-width: auto !important;
}
.js-calendar-graph-svg {
   transform: translate(-8%);
}
</code></pre>

**Comment by ghost** [comment]
<p>Better solution:</p>
<pre><code>body {
   min-width: auto !important;
}
.js-calendar-graph {
   position: relative;
   right: 100px;
}
</code></pre>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/cup">@cup</a>,</p>
<p>Thanks for being here and coming back to post the solutions that worked for you! Much appreciated :slight_smile:</p>

**Comment by ghost** [comment]
<p>I don’t appreciate GitHub breaking their own site. I think your time would be better spent resolving the underlying issue rather than trying to comfort me. Frankly it’s insulting and patronizing</p>

**Comment by ghost** [comment]
<p>Also the issue hasn’t been solved. Some random person marked it solved not me. That’s misleading as only OP should be able to make that decision</p>
